### PR TITLE
Set cluster labels via the Management API: use `kubectl label` command

### PR DESCRIPTION
**Pull request by marians**
This simplifies the MAPI guide by using the `label` command instead of `patch`. Also the placeholders are turned into uppercase letters.

Also I'm removing the conditional for the resource name, as `clusters.cluster.x-k8s.io` works on all providers and is more robust than relying on `clusters`.

### CODE REVIEW

**Review by PrashantR31 — APPROVED** [review]
:+1:

File: src/components/MAPI/clusters/guides/SetClusterLabelsGuide.tsx [hunk 1]
```diff
@@ -65,19 +65,14 @@ const SetClusterLabelsGuide: React.FC<ISetClusterLabelsGuideProps> = ({
...
-              provider === Providers.AWS
-                ? 'clusters.cluster.x-k8s.io'
-                : 'cluster'
-            } ${clusterName} \\
+            label clusters.cluster.x-k8s.io ${clusterName} \\
             --namespace ${clusterNamespace} \\
-            --type merge \\
-            --patch '{"metadata": {"labels": {"foo": "bar"}}}'
+            LABEL=VALUE'
           `}
```
Comment (kuosandys) [inline src/components/MAPI/clusters/guides/SetClusterLabelsGuide.tsx:71]:
```suggestion
            LABEL=VALUE
          `}
```

Comment (marians) [reply src/components/MAPI/clusters/guides/SetClusterLabelsGuide.tsx:71]: Good catch, thanks!

**Review by kuosandys — APPROVED** [review]

**Comment by marians** [comment]
@kuosandys Could you please check whether I have to update a test here? It doesn't appear related to me.

**Pull request merged**